client/src/components/frontend/Layout/Footer.js

```diff
@@ -42,12 +42,14 @@ export default class LayoutFooter extends Component {
   buildContentPages() {
     let pages = [];
     if (this.props.pages) {
-      pages = this.props.pages.map((page) => {
-        return (
-          <Link to={`/browse/page/${page.attributes.slug}`}>
-            {page.attributes.navTitle}
-          </Link>
-        );
+      this.props.pages.forEach((page) => {
+        if (page.showInFooter) {
```

Comment:
This seems wrong to me. Isn't showInFooter an attribute on the page record, and shouldn't this therefore be "page.attributes.showInFooter"?

Did you test this with an actual page record with show_in_footer disabled/enabled? Did it work?

One way you could improve this PR would be to include a test for this element. We have client-side entity factories for our tests now, and you could easily make a page with showInFooter true and one with it as false to confirm we get the direct behavior. Happy to help with this is you'd like me to!